docs/apm/real-user-monitoring.md

```diff
@@ -88,7 +88,7 @@ To configure a RUM HTTP Traces source:
    ```
    * **Probabilistic sampling rate** (optional): Add a **Probabilistic sampling rate** for heavy traffic sites in a decimal value based on percentage, for example, 10% would be entered as `0.1`.
    * **Ignore urls** (optional): Add a list of URLs not to collect trace data from. Supports regex. For example: `/^https:\/\/www.tracker.com\/.*/, /^https:\/\/api.mydomain.com\/log\/.*/`. Please make sure provided URLs are valid JavaScript flavor regexes.
-   * **Custom Tags** (optional): Click **+Add** and enter a key and value for each **Custom Tags** to show in spans from instrumented browsers. For example, click **+Add** and enter a key `deployment.environment` with a value of `production`. This information is saved in the script for `name_of_your_web_service`.
+   * **Custom Tags** (optional): Click **+Add** and enter a key and value for each **Custom Tags** to show in spans from instrumented browsers. This information is saved in the script for `name_of_your_web_service`.
```

Comment:
Would be good to replace removed example with another one, e.g. internal.version=0.1.21

Reply:
So the value would be `internal.version=0.1.21`? What would the Custom Tag  (key) be called?

Reply:
Thread moved to https://github.com/SumoLogic/sumologic-documentation/pull/1414#discussion_r1087813441